ts-typeorm/typeorm/logic.ts

```diff
@@ -10,11 +10,12 @@ export const initialize = async (): Promise<DataSource> => {
    * }
    */ 
     const AppDataSource = new DataSource({
-        type: "postgres",
-        entities: [User],
-        synchronize: true,
-        logging: false,
-    })
+     type: "sqlite",
```

Comment:
is this related to the dependency changes?

Reply:
This is because you need to actually have a postgresql  database running locally to actually test the app if you specify postgres.

Reply:
the runtime will swap out the sqlite bit with the appropriate postgres config at run-time.